### DISCUSSION TITLE
N <= P?

### DISCUSSION THREAD

**Discussion by Amegonzale** [IIC2133-PUC/2024-2#47] [Tarea 0] [answered]
Tengo una duda, cuando dice que un pinguino entra con N puffles puede N ser mayor a la capacidad P? Gracias :3

**Comment by sebacarrascop** [comment, marked as answer]
hola!, no.
<img width="671" alt="image" src="https://github.com/user-attachments/assets/43368d1e-fccf-421a-a230-c997f4f44968">

**Reply by Amegonzale** [reply]
Hola! gracias se me paso xd

**Comment by waknaudt** [comment]
no pero P=NP

**Reply by Amegonzale** [reply]
xD
